rossum/lib/api_client.py

```diff
@@ -537,6 +537,10 @@ def create_hook(
         events: List[str],
         sideload: List[str],
         config: Dict,
+        run_after: List[str] = None,
+        metadata: Optional[Dict] = None,
+        token_owner: Optional[str] = "",
+        test: Optional[str] = None,
```

Comment:
`test` is `Optional[Dict]` above but `Optional[str]` here. I believe `Optional[dict]` is actually correct according to the API reference, though.